.travis.yml

```diff
@@ -15,4 +15,5 @@ notifications:
     on_start: never
 
 after_script:
+  - npm run build
```

Comment:
The linter was catching references in `examples/src/*.js` to `../../lib/index.js` as not existing.  This should fix that.